src/AzureExtension/DevBox/Helpers/WingetConfigWrapper.cs

```diff
@@ -160,14 +166,33 @@ public void Initialize(string configuration)
             _fullTaskJSON = fullTask.ToString();
 
             _units = units;
-            _oldUnitState = Enumerable.Repeat(ConfigurationUnitState.Unknown, units.Count).ToArray();
+            _oldUnitState = Enumerable.Repeat(string.Empty, units.Count).ToArray();
         }
+    }
+
+    private void HandleEndState(TaskJSONToCSClasses.BaseClass response, bool isSetSuccessful = false)
+    {
+        List<ApplyConfigurationUnitResult> unitResults = new();
+        for (var i = 0; i < _units.Count; i++)
+        {
+            var task = _units[i];
+            if (isSetSuccessful || response.Tasks[i].Status == "Succeeded")
+            {
+                unitResults.Add(new(task, ConfigurationUnitState.Completed, false, false, null));
+            }
+            else
+            {
+                unitResults.Add(new(task, ConfigurationUnitState.Completed, false, false, _commonfailureResult));
+            }
+        }
+
+        _applyConfigurationSetResult = new(null, unitResults);
     }
 
     private void SetStateForCustomizationTask(TaskJSONToCSClasses.BaseClass response)
     {
         var setState = DevBoxOperationHelper.JSONStatusToSetStatus(response.Status);
-        _log.Information($"Set Status: {response.Status}");
+        _log.Debug($"Set Status: {response.Status}");
```

Comment:
updated on purpose?

Reply:
Yup.

Reply:
Edit: Realized that this causes the logs to not contain the lines. Reverted.